### DISCUSSION TITLE
Allow pretty printing of json output

### DISCUSSION THREAD

**Discussion by fjuniorr** [manusimidt/py-xbrl#101] [Ideas]
It would be nice if it was possible to pass at least [`indent`](https://docs.python.org/3/library/json.html#:~:text=5%2C%226%22%3A7%7D%5D%27-,Pretty%20printing%3A,-%3E%3E%3E) to [`XbrlInstance.json`](https://github.com/manusimidt/py-xbrl/blob/61a518e7d5efc69647c3f2844540ed78fe39a313/xbrl/instance.py#L289) for pretty printing of JSON output.

**Comment by manusimidt** [comment]
Thanks for the good suggestion!

**Reply by manusimidt** [reply]
see 682e05905ed4d9e6c59de2fe2f087391d0ed14cd

**Reply by fjuniorr** [reply]
Thank you!
